mrmustard/lab/abstract/transformation.py

```diff
@@ -46,6 +46,20 @@ class Transformation:
     r"""Base class for all Transformations."""
     is_unitary = True  # whether the transformation is unitary (True by default)
 
+    def bargmann(self):
+        X, Y, d = self.XYd(allow_none=False)
```

Comment:
I'm a little bit confused about the definition of XYd for Bargmann here. Should they be Abc?

Reply:
we start from X,Y,d like on Serafini's book. Then we output A,B,C by calling the methods `wigner_to_bargmann`